pages/_app.js

```diff
@@ -1,7 +1,9 @@
 import {useEffect} from 'react'
 import PropTypes from 'prop-types'
 import Head from 'next/head'
+
 import 'template.data.gouv.fr/dist/main.css'
+import '../styles/add-calendar.min.css'
```

Comment:
À importer directement dans le composant concerné.

Reply:
Ce fichier permet d'override des classes provenant de la librairie add-to-calendar. 
Je ne vois pas trop comment l'importer au niveau du composant. Je reçois un message d'erreur m'indiquant d'importer la feuille de style dans la page custom App.
Utiliser [cette méthode](https://nextjs.org/docs/basic-features/built-in-css-support#adding-component-level-css) ne fonctionne pas non plus.

Reply:
C'est [Import styles from node_modules](https://nextjs.org/docs/basic-features/built-in-css-support#import-styles-from-node_modules) qui nous intéresse ici.